test/membrane_http_adaptive_stream/integration_test/source_test.exs

```diff
@@ -228,4 +361,19 @@ defmodule Membrane.HTTPAdaptiveStream.Source.Test do
            |> File.read!()
            |> String.starts_with?(expected_prefix)
   end
+
+  defp await_until_file_exists(file_path, wait_s \\ 30) do
```

Comment:
[NIT] It's not precisely waiting up to 30s but rather attempting up to 30 times

Reply:
Why it is not waiting up to 30s?